src/starknet-react/components/FallbackConnector.tsx

```diff
@@ -0,0 +1,105 @@
+type FallbackConnectorProps = {
+  handleClick: () => void
+}
+
+export function FallbackConnector({ handleClick }: FallbackConnectorProps) {
```

Comment:
comment for tsx files, just for context

- why not using svelte (to keep the same lib used in starknetkit)?
- or could we replace svelte with react for our modal

Reply:
I can't use Svelte, this is for the end user who uses starknet-react with starknetkit.

I agree about switching our Svelte files to React, but I'd keep that for some other time, not this PR.